nanome_rmsd/rmsd_menu.py

```diff
@@ -384,18 +384,29 @@ def align_box_button_pressed_callback(button):
             self._plugin.update_content(align_box_button)
             self._plugin.update_content(align_box_text)
 
+        # global <=> local
+        def global_local_button_pressed_callback(button):
+            if self._plugin.args.select == "global":
+                self.update_args("select","local")
+                global_local_button.text.value.set_all("Local")
```

Comment:
isn't the local variable "button" the same as the "global_local_button"? If we can, we should use the local variable.